package.json

```diff
@@ -1,22 +1,39 @@
 {
   "name": "@qld-gov-au/qgds-tokens",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "description": "This repository is configured to sync tokens via Figma.",
-  "main": "index.js",
+  "type": "module",
+  "main": "dist/index.js",
   "scripts": {
     "build-tokens": "style-dictionary build",
```

Comment:
please add a 'prepare' that runs the primary build pipeline so you can import this from git when testing very 'alpha' branches.